Gemfile

```diff
@@ -8,7 +8,7 @@ ruby '3.0.2'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
 gem 'rails', '~> 6.1.4'
 # Use sqlite3 as the database for Active Record
-#gem 'sqlite3', '~> 1.4'
+# gem 'sqlite3', '~> 1.4'
 group :development do
   gem 'sqlite3'
```

Comment:
Why have you changed database? 🤔

Reply:
@kodowicz
Since start I had sqlite 3, but for heroku I will need postgresql as production db, so I decided to change this

Reply:
Do I well understood that your app will work in production(heroku) env with PG, but in dev env, you will work with SQLite?

Never do that, please! You always should replicate your enviroment and it should be always consistent. Even if you have to spent few hours to do that, please migrate whole app to PostgreSQL also for local environment.

Reply:
I've migrated whole app to PostgreSQL :)